flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/DefaultScheduler.java

```diff
@@ -167,9 +171,31 @@ protected DefaultScheduler(
                 jobGraph.getName(),
                 jobGraph.getJobID());
 
+        final Context taskFailureCtx =
+                DefaultFailureEnricherContext.forTaskFailure(
```

Comment:
🤔 Shouldn't we provide more context here? At least the task that the failure originated from?

Reply:
it's fine to create a follow-up ticket for this to get things moving

Reply:
Great idea David! Opened FLINK-32114 to track this